osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -108,7 +108,7 @@ protected override void UpdateItem()
 
             Header.Children = new Drawable[]
             {
-                background = new DelayedLoadWrapper(() => new SetPanelBackground(manager.GetWorkingBeatmap(beatmapSet.Beatmaps.FirstOrDefault()))
+                background = new DelayedLoadWrapper(() => new SetPanelBackground(manager.GetWorkingBeatmap(beatmapSet.Beatmaps.MinBy(b => b.OnlineID)))
```

Comment:
Behavior defined in https://osu.ppy.sh/community/forums/topics/1722422?n=2. Doesn't seem to be documented in wiki https://osu.ppy.sh/wiki/en/Beatmap/Background.

Reply:
The alphabetical sorting is likely a side effect of something (import process order or realm logic) but isn't something we guarantee.

What you have seems fine as long as it's double-checked against stable code. I'll do this if no one else beats me.

Reply:
I tried to have a quick dig in stable source but it looks as if the factor determining what image to show in stable is whatever is hosted at `https://b.ppy.sh/thumb/{beatmap_set_id}.jpg`, and I'm not sure I have the source to check how *that* picks an image, so this is probably one you'll need to poke out of there.

Reply:
Checked BSS code, it looks to be correct. Weakly enforced sort by implicit primary key of query.